src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,26 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { User } from '../../types/User';
+import { getUser } from '../../api';
+import { Todo } from '../../types/Todo';
+interface Props {
+  todo: Todo;
+  onClose: () => void;
+}
+export const TodoModal: React.FC<Props> = ({ todo, onClose }) => {
+  const [user, setUser] = useState<User | null>();
```

Comment:
The `user` state is declared as `User | null`, but it is not explicitly initialized to `null`. It's a good practice to initialize it to `null` to avoid potential undefined behavior.